guides/getting-started-esnext.md

```diff
@@ -0,0 +1,189 @@
+# Getting Started with Aurelia
+
+  Aurelia is an amazing framework that embraces simple clean code without sacrificing power.
+  In this tutorial we'll introduce you to the simplicity of Aurelia through construction of a
+  "Todo" application. You'll see just how clean and simple your application code can be and
+  you will learn the many of the basic concepts and capabilities of Aurelia.
+  
+  Before we start writing code. you will need to get setup with the tools we will be using
+  to create the application. We will be using the aurelia comand line tool to create the project
+  structure. The Aurelia CLI will allow to choose between to popular programming language options: ESNext and typescript. In this tutorial we will be using ESnext as our programming language.  
+  
+## Prerequisites
+
+The first thing that you will need to have installed is node.js it can be downloaded 
+from here https://nodejs.org/. Which can be downloaded from the provided link. After following the installation instructions for node. 
+
+The next tool that will need to be installed is yarn, which can
+be downloaded and installed from here https://yarnpkg.com/lang/en/docs/install/
+
+And finally we need to install the Aurelia CLI tool, which can be installed using the 
+yarn tool we installed in the previous step.
```

Comment:
Maybe include the actual command to install the CLI so people don't need to search for it outside of this guide.